apiserver/logtransfer.go

```diff
@@ -29,7 +29,9 @@ func newMigrationLoggingStrategy(ctxt httpContext, fileLogger io.Writer) Logging
 // Authenticate checks that the user is a controller superuser and
 // that the requested model is migrating. Part of LoggingStrategy.
 func (s *migrationLoggingStrategy) Authenticate(req *http.Request) error {
-	st, err := s.ctxt.stateForMigration(req)
+	// Require MigrationModeNone because logtransfer happens after the
+	// model proper is completely imported.
+	st, err := s.ctxt.stateForMigration(req, state.MigrationModeNone)
```

Comment:
I would have assumed that the mode wouldn't be None, but instead there would be some indicator that the migration was still in progress. Do we not have anything in the target?